### ISSUE TITLE
ldap3

### ISSUE THREAD

**Issue by jayfk** [pyupio/changelogs#57]
ldap3 is currently not parseable.

**Comment by jayfk** [comment]
cc @brianmay

**Comment by cannatag** [comment]
Hi, I'm the ldap3 author, what's the problem with the changelog file? Do you need a specific format to parse?

Bye,
Giovanni

**Comment by jayfk** [comment]
Hi @cannatag. Thanks for keeping a changelog!

> Do you need a specific format to parse?

Not a specific format. Currently, each release is a list item. That makes it hard to parse. It'd help a lot if each release would be prefixed by a `#` or no prefix at all.

I'll put a PR together real quick.
